src/index.html

```diff
@@ -76,7 +76,7 @@ <h2 class="title__heading">
         </div>
         <ul class="about-us__advantages">
           <li class="about-us__advantage advantage">
-            <img src="#" alt="The whole world in our" class="advantage__icon">
+            <img src="images/about-us-1.png" alt="The whole world in our" class="advantage__icon">
```

Comment:
I usually store images like this:
`images/about-us/image-name.png`
Where `image-name` is something that describes the image in few words.

Reply:
It gives easier understanding what's the image is